### ISSUE TITLE
'Swipe to add to queue' like spotify

### ISSUE THREAD

**Issue by Kptomy** [25huizengek1/ViTune#233]
## Feature request: ...

Adding favorite songs in the queue would be even easier if you add an option like 'swipe to add to queue' or 'swipe from left to right'

And also....thumbnail roundness option is not working for me.I don't know why

BTW,   VITUNE IS GREAT
THANKS DEV

**Comment by dtomvan** [comment]
![Screenshot_20240413_163449_Outlook.jpg](https://github.com/25huizengek1/ViTune/assets/51440893/d23886e0-7af8-4589-bf31-4649ec2df1af)

Maybe something like Outlook's swipe options?

**Comment by Kptomy** [comment]
> Maybe something like Outlook's swipe options?

Yes exactly...It would be great to add songs to the queue faster
BTW...THANKS FOR REPLY

**Comment by Kptomy** [comment]
Yes exactly

**Comment by dtomvan** [comment]
What options should we have? Maybe something like:

- Add to queue
- Add to playlist
- Remove from queue/playlist/liked
- Add to favourites
- Hide
- Go to album/artist

Maybe just all common actions on songs that are available...

**Comment by Kptomy** [comment]
1.Add to queue(my priority)...
-Songs can be added to queue after searching for a specific song [right to
left]

2.Add to playlist
-Songs can be added to playlist after searching for a specific song [left
to right]

3.Remove from queue/playlist/liked
-remove from queue already available
-playlist and liked swipe actions are not necessary [I think so]

**Comment by Kptomy** [comment]
This is that option in spotify

**Comment by Kptomy** [comment]
![IMG-20240413-WA0001](https://github.com/25huizengek1/ViTune/assets/152653149/51daf52a-8af2-4e0d-842d-3894fa8b6e5a)
![IMG-20240413-WA0002](https://github.com/25huizengek1/ViTune/assets/152653149/a45d6f81-b109-49d2-b9db-e3d2de9fc74f)

This was the action I was telling about😅

**Comment by genericbetatester** [comment]
It'd be perfect, adding song to queue would be so much faster. Please 😭 @25huizengek1 consider implementing this

**Comment by dtomvan** [comment]
Yes, we will, after we've fixed up the more critical stuff like bugs for example. We are on a very limited time budget here... Thank you for your patience.

**Comment by Kptomy** [comment]
Thanks dev...I can understand
@genericbetatester @dtomvan Love the app and UI

**Comment by divno** [comment]
Play next would also be a good option for swiping

**Comment by siggi1984** [comment]
That would be a mega implementation♥️♥️♥️

**Comment by Kptomy** [comment]
Also check the app named innertune it works fine but vitune has some problem....I have added a new issue.
BTW ....Thanks

**Comment by 25huizengek1** [comment]
ViTune is going into maintenance mode. Only security (or related) issues will be accepted. All current open issues are being closed.
